src/main/java/com/epam/trn/hw6/utils/DataSet.java

```diff
@@ -0,0 +1,89 @@
+package com.epam.trn.hw6.utils;
+
+import com.epam.jdi.tools.DataClass;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataSet extends DataClass<JsonDataReader> {
```

Comment:
What kind of "dataset" is this ? 
Class should be named with reference to it purpose.